clang/docs/ReleaseNotes.rst

```diff
@@ -65,6 +65,9 @@ Clang Python Bindings Potentially Breaking Changes
 - Calling a property on the ``CompletionChunk`` or ``CompletionString`` class
   statically now leads to an error, instead of returning a ``CachedProperty`` object
   that is used internally. Properties are only available on instances.
+- For a single-line ``SourceRange`` and a ``SourceLocation`` in the same line,
```

Comment:
Can you also add a release note to the libclang section about the newly introduced API?

Reply:
Thanks for the reviews, and added 👍🏼